demos/JWST/run_eureka.py

```diff
@@ -19,12 +19,12 @@
 
 if __name__ == '__main__':
 	#s1_meta = s1.rampfitJWST(eventlabel, ecf_path=ecf_path)
-	
-	s2_meta = s2.calibrateJWST(eventlabel, ecf_path=ecf_path)#, s1_meta=s1_meta)
 
-	s3_meta = s3.reduce(eventlabel, ecf_path=ecf_path, s2_meta=s2_meta)
+	s2_meta = s2.calibrateJWST(eventlabel, ecf_path=ecf_path)
 
-	s4_meta = s4.genlc(eventlabel, ecf_path=ecf_path, s3_meta=s3_meta)
+	s3_spec, s3_meta = s3.reduce(eventlabel, ecf_path=ecf_path, s2_meta=s2_meta)
+
+	s4_spec, s4_lc, s4_meta = s4.genlc(eventlabel, ecf_path=ecf_path, s3_meta=s3_meta)
```

Comment:
You will also need to updated these files in the demos/HST/run_eureka_wfc3.py file